tests/TestCase/Routing/RouteBuilderTest.php

```diff
@@ -306,6 +306,23 @@ public function testRedirect()
     public function testPrefix()
     {
         $routes = new RouteBuilder($this->collection, '/path', ['key' => 'value']);
+        $res = $routes->prefix('admin', ['param' => 'value'], function ($r) {
+            $this->assertInstanceOf('Cake\Routing\RouteBuilder', $r);
```

Comment:
Indeed. I think we can open a new issue to track it. Perhaps a considerable amount of code may need this replacement.